low-level/framework/platforms/realstor/realstor_enclosure.py

```diff
@@ -96,6 +96,13 @@ class RealStorEnclosure(StorageEnclosure):
     DATA_FORMAT_JSON = "json"
     FAULT_KEY = "unhealthy-component"
 
+    # Realstor FRUs mapping
+    fru_mapping = {
```

Comment:
why you have added only these frus in fru_mapping list?? what about disk, controller, enclosure etc.

Reply:
so, FRUs through enclosure api '/show/frus'  are :- [POWER_SUPPLY, FAN MODULE, SIDEPLANE]
and in sspl we term as fan, psu, sideplane etc 
That's why this mapping is done.

cli api is not giving info related to 'disk', 'controller' and 'enclosure' FRUs
that I have kept in storage_fru_list>hot_swappable under global config as it is.
so mapping for these frus was not required.

Reply:
>and in sspl we term as fan, psu, sideplane etc

In SSPL also its seen as fan_modules, sideplane_expanders only

btw enclosure is not hot swappable. dont keep 'enclosure' resource in extended configurable fru list also

Reply:
> In SSPL also its seen as fan_modules, sideplane_expanders only

we are using these terms while calling cli apis to get data.
But in sensor resource_type, it is as fan, sideplane etc.

Reply:
> btw enclosure is not hot swappable. dont keep 'enclosure' resource in extended configurable fru list also

ok removed from default list